packages/validator/src/capability.js

```diff
@@ -635,29 +635,52 @@ class AndMatch {
 }
 
 /**
- * Parses capability `source` using a provided capability `parser`. By default
- * invocation parsing occurs, which respects a capability schema, failing if
- * any non-optional field is missing. If `optional` argument is `true` it will
- * parse capability as delegation, in this case all `nb` fields are considered
- * optional.
+ * @template {string} T
```

Comment:
```suggestion
 * Takes an ability (can field of the capability) and an ability (wildcard) pattern
 * and returns back ability if pattern matches or `null` if it does not otherwise. 
 * @example
 * ```js
 * matchAbility('store/add', 'store/list') // => null
 * matchAbility('store/add', 'store/*') // => 'store/add'
 * matchAbility('store/add', '*') // => 'store/add'
 * matchAbility('store/add', 'store/') // => null
 * matchAbility('store/add', 'store/a*') // => null
 * ```
 * @template {API.Ability} T
```